### ISSUE TITLE
Add subctl validate test

### ISSUE THREAD

**Issue by manosnoam** [manosnoam/ocp-multi-cluster-tester#40]
@pinikomarov  Can you look at the new feature to test: subctl validate ?
https://submariner.io/operations/deployment/subctl/

**Comment by pinikomarov** [comment]
sure , but validate is not on that doc yet right ?

**Comment by manosnoam** [comment]
yes, it's being developed for 0.9 (ask @Sridhar Gaddam ***@***.***>
)

**Comment by sridhargaddam** [comment]
Its under active development for 0.9 - https://github.com/submariner-io/submariner-operator/issues/1126

**Comment by manosnoam** [comment]
@pinikomarov the feature is ready for test:
https://submariner.io/operations/deployment/subctl/#validate

**Comment by pinikomarov** [comment]
ack

**Comment by pinikomarov** [comment]
Hi @manosnoam can you check this patch out  ? 

https://github.com/manosnoam/ocp-multi-cluster-tester/pull/44

Thanks
